### PR TITLE
@types/node: add missing properties to Http2Session

### PR DESCRIPTION
**Pull request by murgatroid99**
Add "connecting" and "ping" properties to Http2Session

Recreating #27207 because it got closed.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://nodejs.org/api/http2.html>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by murgatroid99** [comment]
This is kind of ridiculous. The CI build failure has problems in two different packages that appear to have been introduced since I last tried to make this PR two weeks ago.

**Comment by Flarna** [comment]
I can't to more then others - just try to understand where the problem is and create PRs to fix them.

But this time @andy-ms did this already: I found #27542 which should fix the issue in redux-form/v4 and #27541 which should fix the issue in react-sortable-tree.

Let's hope that no new issue are introduced till they get merged. Unfortunately CI is not 100% water prove because of several reasons:
* dependencies to other repos via various package.json files and typescript/lint
* no CI to verify that removing a package doesn't break others depending on it

Sometimes it works fine for weeks or even month - but every now and then it's a real nightmare to get something merged because of unrelated CI fails.

File: types/node/index.d.ts [hunk 1]
```diff
@@ -7181,13 +7181,16 @@ declare module "http2" {
...
+        readonly connecting: boolean;
         destroy(error?: Error, code?: number): void;
         readonly destroyed: boolean;
         readonly encrypted?: boolean;
         goaway(code?: number, lastStreamID?: number, opaqueData?: Buffer | DataView | NodeJS.TypedArray): void;
         readonly localSettings: Settings;
         readonly originSet?: string[];
         readonly pendingSettingsAck: boolean;
+        ping(callback: (err: Error | null, duration: number, payload: Buffer) => void): boolean;
+        ping(payload: Buffer | DataView /*| TypedArray */, callback: (err: Error | null, duration: number, payload: Buffer) => void): boolean;
```
Comment (Flarna) [inline types/node/index.d.ts]: Please use `NodeJS.TypedArray` like in `goaway()`.

**Review by Flarna — CHANGES_REQUESTED** [review]
please add `TypedArray`

File: types/node/index.d.ts (hunk 1 above)
Comment (murgatroid99) [reply types/node/index.d.ts]: done

**Review by Flarna — APPROVED** [review]

**Pull request merged**